data-prepper-core/src/test/java/org/opensearch/dataprepper/peerforwarder/PeerForwarderConfigurationTest.java

```diff
@@ -159,7 +169,7 @@ void test_with_acm_should_create_PeerForwarderConfiguration_object_even_with_nul
             TestDataProvider.INVALID_PEER_FORWARDER_WITH_CLOUD_MAP_WITHOUT_NAMESPACE_NAME_CONFIG_FILE,
             TestDataProvider.INVALID_PEER_FORWARDER_WITH_CLOUD_MAP_WITHOUT_REGION_CONFIG_FILE,
             TestDataProvider.INVALID_PEER_FORWARDER_WITH_DNS_WITHOUT_DOMAIN_NAME_CONFIG_FILE,
-            TestDataProvider.INVALID_PEER_FORWARDER_WITH_SSL,
+//            TestDataProvider.INVALID_PEER_FORWARDER_WITH_SSL
```

Comment:
Should this be removed? If so, let's remove the line entirely.

Reply:
These tests should be removed. They are invalid